src/scripts/helpers/backbone/views/base.coffee

```diff
@@ -77,11 +77,12 @@ define (require) ->
         historyPage = linksHelper.getCurrentPathComponents().page
 
       if @pageTitle
-        document.title = @pageTitle + settings.titleSuffix
+        pageTitle = @pageTitle.replace(/<\/?[^>]+(>|$)/g, '') # Strip tags for baked HTML titles
+        document.title = pageTitle + settings.titleSuffix
```

Comment:
A common way to strip html tags is to create a scratch div element, set its innerHTML to the HTML you want to strip, then take its `textContent`.